tutorial_query_expansion.md

```diff
@@ -2,20 +2,21 @@
 
 ## Let's get started
 
-This tutorial will show you how to enable the query expansion feature which allows you to increase the recall for query terms with multiple results,
-especially long tail queries.
+This tutorial will show you how to enable the query expansion feature which allows you to **increase the efficiency for search** for ambiguous or long-tail* query terms. 
 
 If the query expansion is disabled, only the exact search query is used, even if the total number of search result is zero.
 
-Enable query expansion feature and let Google Retail Search build an automatic query expansion.
+Enable query expansion feature and let Google Retail Search build an **automatic query expansion**.
 
-You can also pin unexpanded products, so they will always be at the top of search results, followed by those that come via expansion.
-
-This is a really useful feature that helps you enhance a customer experience. So let's look at it closely.
+You can also **pin unexpanded products**, so they will always be at the top of search results, followed by those that come via expansion.
 
+This is a really useful feature that helps you **enhance a customer experience**. So let's look at it closely.
```

Comment:
Rephrase:

This useful feature helps you to **enhance a customer experience**. Let's look at it closely.